endpoints/lib/vboxwebService-5.1.wsdl

```diff
@@ -5,7 +5,7 @@
   Generator: src/VBox/Main/webservice/websrv-wsdl-service.xsl
 -->
 <definitions xmlns:interface="urn:vbox" xmlns:soap="http://schemas.xmlsoap.org/wsdl/soap/" xmlns:vbox="http://www.virtualbox.org/" xmlns="http://schemas.xmlsoap.org/wsdl/" name="VirtualBox" targetNamespace="http://www.virtualbox.org/Service">
-  <import location="vboxweb-5.0.wsdl" namespace="http://www.virtualbox.org/"/>
+  <import location="vboxweb.wsdl" namespace="http://www.virtualbox.org/"/>
```

Comment:
Yes, you are right. Sorry for this. 
I copied the generated files from virtualbox and while testing I had both files (vboxweb.wsdl and vboxweb-5.1wdsl) in the directory.